src/main/java/seedu/address/model/tag/UniqueTagList.java

```diff
@@ -73,6 +66,10 @@ public void setTags(List<Tag> tags) {
         tagArrayList.setAll(tags);
     }
 
+    /**
+     * Returns an unmodifiable list of tags.
+     * @return The list of unmodifiable tags.
```

Comment:
I believe that there should not be periods `.` at the end of `@return` javadocs.

Reply:
Ok. I made the changes.